### ISSUE TITLE
Visuelle Darstellung von Inzidenzen pro Altersgruppe

### ISSUE THREAD

**Issue by HerrMeneutik** [corona-warn-app/cwa-wishlist#630]
## Feature description

Eine visuelle Darstellung von Inzidenzen im Bundesgebiet bzw auf Bundeslandebene nach Altersgruppen aufgeschlüsselt. Interessant wäre dann noch die Anzeige wieviel Prozent der gerade angezeigten Gruppe (6-14 Jahre NRW z.b.) bereits vollständig geimpft sind. 

## Problem and motivation

Ich halte eine niederschwellige und zugängliche Bereitstellung dieser Daten für gesellschaftlich relevant. 

## Is this something you're interested in working on

Wenn ich etwas beitragen kann, sehr gerne.

---
Internal Tracking ID: [EXPOSUREAPP-11566](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11566)

**Comment by MikeMcC399** [comment]
@HerrMeneutik 

Unter https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/Gesamt.html wird z. Zt. Donnerstags die Inzidenzzahlen nach Altersgruppen aufgeschlüsselt für die Bundesrepublik veröffentlicht.

Die derzeit letzte Veröffentlichung ist vom 19.08.2021 https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/Wochenbericht/Wochenbericht_2021-08-19.pdf?__blob=publicationFile Seite 4, Abbildung 1: Darstellung der 7-Tage-Inzidenz der COVID-19-Fälle in Deutschland nach Altersgruppe und Meldewoche (n=3.604.760 Fälle
mit entsprechenden Angaben in den Meldewochen 33/2020 bis 32/2021; Stand 17.08.2021, 0:00 Uhr).

Auf Seite 16 wird über das Impfen Digitales Impfquotenmonitoring (DIM) berichtet.

**Comment by MikeMcC399** [comment]
Also refer to https://impfdashboard.de/ for statistics regarding vaccination.

**Comment by HerrMeneutik** [comment]
Yes thank you. I think having this information right within the app, people can easily access it and evaluate their personal risk of exposure. What do you think?

**Comment by larswmh** [comment]
Thanks for your feature request @HerrMeneutik. We have created an internal ticket for it and will raise this topic internally.
Internal Tracking ID: [EXPOSUREAPP-11566](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11566)

---
Corona-Warn-App Open Source Team

**Comment by MikeMcC399** [comment]
@larswmh / @HerrMeneutik 

The requested "Visuelle Darstellung von Inzidenzen pro Altersgruppe" is not available directly in the app. It is however linked to by the CWA website.

1. Open https://www.coronawarn.app/de/analysis/ "Corona-Warn-App (CWA): Kennzahlen"
2. Scroll down to bottom "Weitere Dashboards"
3. Select RKI: Covid-19-Trends (DE)
4. Scroll down to "Altersgruppen"

Or go directly to link [Covid-19-Trends in Deutschland im Überblick](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/COVID-19-Trends/COVID-19-Trends.html?__blob=publicationFile#/home) and scroll down to "Altersgruppen".

![image](https://user-images.githubusercontent.com/66998419/150988948-eb5fc262-eaaa-42c1-ad54-f76ea17f047c.png)

**Comment by MikeMcC399** [comment]
Um die Anzeige "Altersgruppen" in der App zu erhalten:

1. Auf der Status-Seite der Corona-Warn-App herunterscrollen bis zum Bereich "Häufige Fragen" und darauf tippen.
2. Im Menu rechts oben, "Analysis" auswählen.
3. Auf der Seite  "Corona-Warn-App (CWA): Kennzahlen" bis zum Link "RKI: Covid-19-Trends (DE)" nach unten scrollen.
4. Auf diesen Link tippen.
5. Bis zum "Altersgruppen" herunterscrollen.

Insofern ist diese Anfrage bereits realisiert.
